manageiq-operator/api/v1alpha1/helpers/miq-components/httpd.go

```diff
@@ -599,7 +600,30 @@ func HttpdDbusAPIService(cr *miqv1alpha1.ManageIQ, scheme *runtime.Scheme) (*cor
 	return service, f
 }
 
-func TLSSecret(cr *miqv1alpha1.ManageIQ) (*corev1.Secret, error) {
+func ManageTlsSecret(cr *miqv1alpha1.ManageIQ, client client.Client, scheme *runtime.Scheme) (*corev1.Secret, controllerutil.MutateFn, error) {
```

Comment:
Why did this change?

Reply:
It was very complicated to keep using `createk8sResIfNotExist` after the upgrade. https://github.com/ManageIQ/manageiq-pods/pull/831/commits/df565c2221b8aaf71e502ef0d79290f0fd9315fb